tools/add_header

```diff
@@ -172,7 +175,7 @@ def main(arguments):
 
             sys.exit(0)
 
-        if str(MyFile).endswith('.py'):
+        if str(MyFile).endswith('.py') or MyFile.split('/')[0] == "bin":
```

Comment:
I think this needs a more sophisticated check, making the path absolute and then checking the parent directory of `MyFile`. I sometimes run this script on absolute paths or from within the bin directory and different results would be weird in those cases.